app/utilities.py

```diff
@@ -1,6 +1,28 @@
 from typing import Dict
+from app.data import MongoDB
 
+db = MongoDB("UnderdogDevs")
 
 def dict_to_str(data: Dict) -> str:
     """Convert dictionaries into easy to read strings."""
     return "\n" + "\n".join(f"{k}: {v}" for k, v in data.items())
+
+
+def financial_aid_gen(profile_id):
+    '''This is a function that queries the database
+    for variables attached to a mentees profile id and applies
+    that information into a forumla that returns financial aid
+    probability'''
+
+    try:
+        profile = db.read('Mentees', {"profile_id": profile_id})
+        E_l_dict = {"Beginner" : 0, "Intermediate" : 1, "Advanced" : 2, "Expert" : 3}
```

Comment:
2 quick things:
- PEP8 uses snake_case for most things, except class names, but even then, it can still be snake_case in certain circumstances. [Food for Thought](https://peps.python.org/pep-0008/#class-names)
- There is also a logical error due to beginner value of 0: pow(9.9*(experience_level), -1), which will result in a ```ZeroDivisionError```

Reply:
Thank you for pointing that out. Those values should be 1-5 anyway, and not 0-4.

Reply:
We removed the Try, Except. We now have a HTTPException in the FastAPI to let a user know if an ID is not in the DB.

Reply:
Fixed the formula and updated pep8 guidelines.